js/client-teambuilder.js

```diff
@@ -1245,6 +1254,39 @@
 			this.save();
 		},
 
+		setNotesBlur: function (e) {
+			var textarea = $(e.currentTarget);
+			if (!$.trim(textarea.val())) textarea.parent().hide();
+			if (textarea.val().indexOf('|') >= 0 || textarea.val().indexOf(']') >= 0 || textarea.val().indexOf('\t') >= 0) {
+				app.addPopupMessage("Unfortunately, set notes can't contain tabs or the characters | and ], since they're used for storing teams.");
```

Comment:
If the main problem is _other_ sites storing large amounts of data locally, then the increase in team size from set notes is unlikely to be what pushes it over the storage limit, isn't it? Unless that limit was only a megabyte or two to begin with.
>Hm, I think overall this is probably not going to be a huge problem

Does that mean this PR can proceed?